pkg/operator/ceph/csi/controller.go

```diff
@@ -332,7 +332,7 @@ func (r *ReconcileCSI) reconcile(request reconcile.Request) (reconcile.Result, e
 		}
 	}
 
-	if !EnableCSIOperator() {
+	if !EnableCSIOperator() && !disableCSI {
 
```

Comment:
```suggestion
```

nit, not related to your change but if you could remove this empty line

Reply:
ya removed